Client/Frontend/Brave Rewards/OptIn/BraveTalkOptInSuccessView.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import BraveUI
+import Shared
+import BraveShared
+
+extension BraveTalkOptInSuccessViewController {
+    class View: UIView {
+        
+        private let image = UIImageView(image: #imageLiteral(resourceName: "optin_check_circle")).then {
+            $0.contentMode = .scaleAspectFit
+        }
+        
+        private let title = UILabel().then {
+            $0.text = Strings.Rewards.braveTalkRewardsOptInSuccessTitle
+            $0.font = .systemFont(ofSize: 20)
```

Comment:
Use preferred fonts/scaled fonts and set `adjustsFontForContentSizeCategory` to true